### PR TITLE
k8s

### LINKED ISSUE #19: k8s

### PR DESCRIPTION
**Pull request by eyakauleva**
close #19

### CODE REVIEW

File: src/infra/execute.sh [hunk 1]
```diff
@@ -14,4 +14,4 @@ kubectl apply -f app-configmap.yaml
 kubectl apply -f app-service.yaml
 kubectl apply -f app-deployment.yaml
 
-#minikube service user-service
\ No newline at end of file
+minikube service user-service
```
Comment (h1alexbel) [inline src/infra/execute.sh:17]: since you are using `minikube` here, it's better to name script `local.sh` or `minikube.sh`

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/infra/zookeeper-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+kind: Deployment
```
Comment (h1alexbel) [inline src/infra/zookeeper-deployment.yaml:1]: zookeeper should be a stateful set

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/infra/zookeeper-deployment.yaml (hunk 2 above)
Comment (eyakauleva) [reply src/infra/zookeeper-deployment.yaml]: fixed 89b30ff

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@eyakauleva cheers

**Pull request merged**